resources/views/emails/mship/account/welcome.blade.php

```diff
@@ -2,73 +2,57 @@
-</p>
+<p><strong>TeamSpeak</strong> - We use TeamSpeak for voice communication. This includes coordination whilst controlling, one-to-one mentoring sessions, group seminars and general chat.</p>
+<div style="margin-bottom: 2em;"><a href="{{route('site.community.teamspeak')}}" class="btn btn-primary">Join us on TeamSpeak</a></div>
 
-<p>
-    If any details are incorrect, or you have any concerns, please
-    contact {!! link_to('mailto:community@vatsim.uk', 'our community department') !!} who will be able to help you
-    further.
-</p>
+<p><strong>Community Forum</strong> - Our community forum is a place for text discussion; express your interest in events, share screenshots of your latest flight and keep up to date on the latest news.</p>
+<div style="margin-bottom: 2em;"><a href="https://community.vatsim.uk" class="btn btn-primary">Take a look around our Forum</a></div>
 
-<h2>What next?</h2>
+<p><strong>Division Website</strong> - Our website hosts all the information you might need whilst a member of the network. There's lot to digest, and we'll help you find what you need!</p>
+<div style="margin-bottom: 0.5em;"><a href="{{route('site.home')}}" class="btn btn-primary">Visit our website</a></div>
+<div style="margin-bottom: 2em;"><a href="{{route('dashboard')}}" class="btn btn-primary">Check your personal details</a></div>
 
+
+<h2>✈️️ Training</h2>
 <p>
-    If you're interested in flying, VATSIM UK offers multiple pilot training courses:
-    {!! link_to_route('site.pilots.landing') !!}
+    If you're interested in flying, VATSIM UK offers multiple pilot training courses. Browse our current offering at
+    {!! link_to_route('site.pilots.landing') !!}.
 </p>
 
 <p>
-    If you're interested in providing ATC:
-    {!! link_to_route('site.atc.newController') !!}
+    If you're interested in learning about Air Traffic Control, with the goal of controlling on the VATSIM network, check out
+    {!! link_to_route('site.atc.newController') !!}.
 </p>
 
+<h2>❓ Getting help</h2>
 <p>
-    For both Pilot and ATC training our online system will allow you to book mentoring (once you are enrolled as per the above links):
-    {!! link_to("https://cts.vatsim.uk/", "https://cts.vatsim.uk/") !!}
+    We recommend making use of our <a href="{{route('discord.show')}}">Discord</a>, <a href="https://community.vatsim.uk">community forum</a>, or <a href="{{route('site.community.teamspeak')}}">TeamSpeak</a> to get help from members and division staff.
 </p>
-
 <p>
-    The UK TeamSpeak Server and Forum - these are used for voice and written communications for the entire UK community. Feel free to join the community and ask questions and meet others:
-    <li> TeamSpeak - {!! link_to_route('site.community.teamspeak') !!}</li>
-    <li> Forum - {!! link_to("https://community.vatsim.uk/", "https://community.vatsim.uk/") !!}</li>
+    You'll be in good company, although there is lots to learn, there are plenty of people ready to support you.
```

Comment:
This doesn't sound quite right, maybe:

You'll be in good company, and although...

Apart from that all good here!

Reply:
Changed it to `Although there is a lot to learn, you will be in good company, surrounded by like-minded simming enthusiasts ready to help you get started.`